docs/source/getting_started/index.rst

```diff
@@ -227,7 +226,7 @@ Creating the First Query
 
 Now let's check if ``coolcoins`` were successfully transferred from
 ``admin@test`` to ``test@test``. Choose ``2. New query
-(qry)``. ``7. Get Account's Assets (get_acc_ast)`` can help you to check if
+(qry)``. ``8. Get Account's Assets (get_acc_ast)`` can help you to check if
```

Comment:
Those numbers (7/8) can vary from build to build - it is ok. 
There is no much need to keep them up to date.